src/redux/books/books.js

```diff
@@ -0,0 +1,8 @@
+const books = () => (
+  <form>
+    <input type="text" className="title" placeholder="title" />
+    <input type="text" className="author" placeholder="author" />
+    <button type="submit">submit</button>
+  </form>
```

Comment:
- [ ] The form should be rendered in the `CreateNewBook` component as suggested in the previous review. Kindly fix it.